vars/default.yml

```diff
@@ -1,2 +1,5 @@
 ---
-certbot_package: certbot
+certbot_package:
```

Comment:
Maybe certbot_package_extras variable can be added to install additional packages?
I'll add it in my other commit.